### PR TITLE
Added Handle Hide delay

### PR DESCRIPTION
**Pull request by tom5079**
(cherry picked from commit 2bf12cae5977220e8eae2831cac21d52587da78c)

### CODE REVIEW

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt [hunk 1]
```diff
@@ -324,6 +329,9 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
                 (loadDrawableFromAttributes(R.styleable.RecyclerViewFastScroller_handleDrawable)
                     ?: ContextCompat.getDrawable(context, Defaults.handleDrawableInt))
 
+            hideHandleAfter =
```
Comment (shahsurajk) [inline recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: let change this to: 
```suggestion
            handleVisibilityDuration =
```
let's also change the style attribute

**Review by shahsurajk — CHANGES_REQUESTED** [review]
looks great, just one nit

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt [hunk 2]
```diff
@@ -126,6 +127,7 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
         const val DEFAULT_ANIM_DURATION: Long = 100
         const val DEFAULT_POPUP_VISIBILITY_DURATION = 200L
         const val hasEmptyItemDecorator: Boolean = true
+        const val hideHandleAfter: Int = 0
```
Comment (shahsurajk) [inline recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: let's change this as well and use uppercase for constant values 
```suggestion
        const val DEFAULT_HANDLE_VISIBILITY_DURATION: Int = 0
```

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt [hunk 3]
```diff
@@ -126,6 +127,7 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
         const val DEFAULT_ANIM_DURATION: Long = 100
         const val DEFAULT_POPUP_VISIBILITY_DURATION = 200L
         const val hasEmptyItemDecorator: Boolean = true
```
Comment (shahsurajk) [inline recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt:129]:
```suggestion
        const val DEFAULT_HAS_EMPTY_ITEM_DECORATOR: Boolean = true
```

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt [hunk 4]
```diff
@@ -126,6 +127,7 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
...
+        const val hideHandleAfter: Int = 0
         const val trackMargin: Int = 0
```
Comment (shahsurajk) [inline recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt:131]: uppercase constants
```suggestion
        const val DEFAULT_TRACK_MARGIN: Int = 0
```

**Review by shahsurajk — CHANGES_REQUESTED** [review]

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt (hunk 2 above)
Comment (tom5079) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt]: I thought lowercase values are for the default property values. I think it's a different issue and handled separately.

File: recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt (hunk 3 above)
Comment (tom5079) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt:129]: I didn't even add this part?

Comment (shahsurajk) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt:129]: can you please update it. looks like a mistake from someone from the team then. but it definitely shouldnt be camelcase

Comment (tom5079) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt:129]: Should I change all the properties in `Defaults` then? Changing part of them only makes code inconsistent. How about opening up separate PR or commit and fix it there? It's not only these three that I have to fix and it is completely unrelated to this PR.

Comment (shahsurajk) [reply recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt:129]: agreed. let me make a different issue on the same.

**Review by shahsurajk — APPROVED** [review]

**Pull request merged**